config/nxf_call_cnv.config

```diff
@@ -0,0 +1,25 @@
+includeConfig 'nxf.config'
+
+env {
+  CMD_MOSDEPTH="apptainer exec --no-mount home --bind \${TMPDIR} ${APPTAINER_CACHEDIR}/mosdepth-0.3.3.sif mosdepth"
+  CMD_SPECTRE="apptainer exec --no-mount home --bind \${TMPDIR} ${APPTAINER_CACHEDIR}/spectre-0.2.0.sif spectre"
+}
+
+params {
+  cnv {
+    spectre {
+      GRCh38 {
+        metadata = "${projectDir}/resources/GRCh38/spectre.metadata.mdr"
+        blacklist = ""
+      }
+    }
+  }
+}
+
+process {
+  withLabel: 'spectre_call' {
+    cpus = 4
+    memory = '8GB'
+    time = '5h'
```

Comment:
`nanopore` test case ran in `1m 44s`? `5h` is too much? at a minimum stick to the `4h` default?